### DISCUSSION TITLE
"Resolving Conflicts" Practices to "Preserve Latest Changes"

### DISCUSSION THREAD

**Discussion by MastanSayyad** [Priyaaa1/StartConnect-Hub#514] [Ideas]
Hey @Priyaaa1 I have noticed that During the process of resolving conflicts, there have been instances where contributors have removed or overwritten the latest changes made to the project. This issue is causing the following problems:

- Previously merged PRs are being undone or altered, leading to inconsistencies in the project.
- The most recent changes and improvements to the website are being lost

### Examples

- **Example 1:**  In one PR someone fixed "How it works" navigations in footer but after that some other contributor made some changes in formatting of footer and the changes made by the previous contributor got undone (You can see "How it works" page link is now not present in footer)
- **Example 2:** The light mode dark mode functionality was introduce by someone who fixed the text readabilty and toggle issue and which was working fine but as you can see the dark mode functionality is not working properly it only changes theme of some parts while other remains the same.
- **Example 3:** As you know I have added feedback page and put the navigation links in both navbar and footer but the now you can see the link is only in footer not in navbar. it happened due to latest merged PR changes

There may be many more issues like this, these are the ones I have noticed.

**The primary focus should be on preserving the latest changes while integrating the contributor’s work seamlessly.**

### Recommended Guidelines for Contributors:
1. **Understand the Conflict:** Before resolving a conflict, review the changes in the conflicting files to understand what the latest changes are.
2. **Preserve Latest Changes:** Ensure that the latest updates and changes are preserved. Only integrate your changes around these updates without removing them.
3. **Communicate:** If unsure about how to resolve a particular conflict, communicate with the project admin or the contributor who made the recent changes.
4. **Test:** After resolving conflicts, thoroughly test the affected areas to ensure that the latest changes are intact and functional.

I would suggest you to tell the contributors to not change, remove or undone the latest changes while resolving conflicts.

Thank You!

**Comment by Priyaaa1** [comment]
@MastanSayyad yeah I'll look into this, thankyou for updating me about this !
And I've removed some elements from the navbar as I think those should not be there in the navbar, one of those was the Feedback page. 'How it works' page was removed from the footer by someone else which I have added now.
